Makefile

```diff
@@ -163,7 +163,7 @@ clean: ## Clean.
 export KUBEBUILDER_ASSETS=$(PROJECT_DIR)/bin
 
 .PHONY: test
```

Comment:
If the mocks need updating, they need to be updated prior to linting and other commands.